### PR TITLE
add log4perl logging to server web app

### PR DESCRIPTION
**Pull request by haarg**
The scripts use their own configuration for now, although preferrably we
will merge them together.

### CODE REVIEW

File: lib/MetaCPAN/Server/Controller/User.pm [hunk 1]
```diff
@@ -5,6 +5,7 @@ use warnings;
 
 use DateTime;
 use Moose;
+use Log::Log4perl::MDC;
```
Comment (ranguard) [inline lib/MetaCPAN/Server/Controller/User.pm]: ? - why is this needed here?

Comment (haarg) [reply lib/MetaCPAN/Server/Controller/User.pm]: Later in the file it is used to attach some extra request data to the logging.
